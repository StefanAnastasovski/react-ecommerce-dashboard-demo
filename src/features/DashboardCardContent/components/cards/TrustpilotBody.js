import React from "react";

import { Box } from "@mui/material";

import {
  LINK_N_TEXT_FONT_SIZE,
  PRIMARY_GREEN_COLOR_ACTIVE,
  WHITE_COLOR,
  FONT_WEIGHT_700,
} from "../../../../data/constants";

import Paragraph from "../../../../components/Typography/Paragraph";

const TrustpilotBody = () => {
  const fontSize = {
    fontSize: LINK_N_TEXT_FONT_SIZE,
  };
  const whiteText = {
    color: WHITE_COLOR,
  };

  const highlightedText = {
    color: PRIMARY_GREEN_COLOR_ACTIVE,
  };

  return (
    <>
      <Paragraph {...{ ...whiteText, ...fontSize }} my={2}>
        Show us your love by leaving a
        <Box component="span" {...highlightedText} px={0.5}>
          positive
        </Box>
        review on trust pilot and receive the extension of
        <Box
          component="span"
          {...whiteText}
          fontWeight={FONT_WEIGHT_700}
          pl={0.5}
        >
          50 additional products.
        </Box>
      </Paragraph>
    </>
  );
};

export default TrustpilotBody;
