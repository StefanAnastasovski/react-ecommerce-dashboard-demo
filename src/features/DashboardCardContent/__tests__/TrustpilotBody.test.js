import React from "react";
import { shallow } from "enzyme";

import TrustpilotBody from "../components/cards/TrustpilotBody";

describe("<TrustpilotBody />", () => {
  describe("render()", () => {
    it("Shallow rendering - MatchSnapshot", () => {
      const component = shallow(<TrustpilotBody />);
      expect(component).toMatchSnapshot();
    });
  });
});
