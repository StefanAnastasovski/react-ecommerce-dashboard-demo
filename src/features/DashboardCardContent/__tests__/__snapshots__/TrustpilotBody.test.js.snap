// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TrustpilotBody /> render() Shallow rendering - MatchSnapshot 1`] = `ShallowWrapper {}`;
